test/azure/RspecTests/resource_inheritance_spec.rb

```diff
@@ -0,0 +1,58 @@
+# encoding: utf-8
+
+$: << 'RspecTests/Generated/azure_resource_inheritance'
+
+require 'rspec'
+require 'generated/resource_inheritance'
+
+include AzureResourceInheritanceModule
+
+describe 'ResourceInheritance' do
```

Comment:
should this be renamed, as it's not testing inheritance for resource anymore? but validating that all models are generated for any inheritance relationship?